src/scaler/spreadscaler/link.rs

```diff
@@ -219,7 +222,14 @@ impl<S: ReadStore + Send + Sync, L: LinkSource> LinkScaler<S, L> {
             provider_link_name.unwrap_or_else(|| DEFAULT_LINK_NAME.to_string());
         // NOTE(thomastaylor312): Yep, this is gnarly, but it was all the information that would be
         // useful to have if uniquely identifying a link scaler
-        let id = format!("{LINK_SCALER_TYPE}-{model_name}-{provider_link_name}-{actor_reference}-{provider_reference}");
+        let id = if let Some(linkscaler_values) = &values {
+            // When values are present, we want to include them in the ID as well
+            let linkscaler_values_hash = compute_linkscaler_values_hash(linkscaler_values);
+            format!("{LINK_SCALER_TYPE}-{model_name}-{provider_link_name}-{actor_reference}-{provider_reference}-{linkscaler_values_hash}")
+        } else {
+            format!("{LINK_SCALER_TYPE}-{model_name}-{provider_link_name}-{actor_reference}-{provider_reference}")
+        };
```

Comment:
I am a little concerned about this unique ID getting large but I don't know if that would cause issues 🤔

Reply:
may be a separate ticket for this (if/when it becomes a problem)?

Reply:
We could always start hashing this text if we need to. But I'm ok to yolo for now